File: server/Timelogger/Entities/TimeRegistration.cs
using System;
using System.Text.Json.Serialization;

namespace Timelogger.Entities
{
    public class TimeRegistration
    {
        public int TimeRegistrationId { get; set; }
		public string Title { get; set; }
		public string Description { get; set; }
		public int TimeSpent { get; set; }
    }
}

File: client/src/app/models/timeregistration.ts
export class TimeRegistration {

    timeRegistrationId: number;
    title: string;
    description: string;
    timeSpent: number;

    constructor(timeRegistrationId: number, title: string, description: string, timeSpent: number) {
        this.timeRegistrationId = timeRegistrationId;
        this.title = title;
        this.description = description;
        this.timeSpent = timeSpent;
    }
  }